.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+index.js
+src/application.js
```

Comment:
I find this rather ugly, because it makes it difficult to clean files generated by the build step. I am proposing to configure `outDir` as `dist` in `tsconfig.json` and then git-ignore entire `dist` folder.